app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,22 @@
+@import "font-awesome";
+.top-right-btn {
+  
+    position: absolute;
+    top: 75px;
+    right: 20px;
+    padding-right: 50px;
+  }
+.image {
+  width: 1547px;
+}
+.card{
+  max-width: 72rem;
+  padding-top: 60px;
+}
+.table{
+  padding-left: 2100px;
+}
+.img {
+  max-width: 38%;
+  width: 367px;
+}
```

Comment:
Add a blank line at the end of the file and include a blank line after each class definition.